pkg/operator/encryption/migration_controller.go

```diff
@@ -198,46 +196,73 @@ func (c *migrationController) migrateKeysIfNeededAndRevisionStable() (resetProgr
 			continue
 		}
 
-		// storage migration takes a long time so we expose that via a distinct status change
-		if err := c.setProgressing(strings.Title(groupToHumanReadable(gr))+strings.Title(gr.Resource), "migrating resource %s.%s to new write key", groupToHumanReadable(gr), gr.Resource); err != nil {
-			return false, err
-		}
-
 		if needsMigration(writeSecret, gr) {
-			migrationErr := c.runStorageMigration(gr)
-			errs = append(errs, migrationErr)
-			if migrationErr != nil {
-				continue
+			// storage migration takes a long time so we expose that via a distinct status change
+			if err := c.setProgressing(strings.Title(groupToHumanReadable(gr))+strings.Title(gr.Resource), "migrating resource %s.%s to new write key", groupToHumanReadable(gr), gr.Resource); err != nil {
+				return false, err
```

Comment:
we count continue, migration is more important than setting a status, wdyt?

Reply:
we will continue anyway, 2min later. Both ways would be ok I guess, either wait just continue here. Do you have a strong preference?

Reply:
would prefer to continue stright away but please add a comment if you don't mind.

Reply:
Don't want to risk the green tests now. But put it on the list for TODOs. Have no strong opinion.